clients/python/client/osparc/_studies_api.py

```diff
@@ -36,3 +58,53 @@ def create_study_job(self, study_id, job_inputs, **kwargs):
     def clone_study(self, study_id, **kwargs):
         kwargs = {**kwargs, **ParentProjectInfo().model_dump(exclude_none=True)}
         return super().clone_study(study_id, **kwargs)
+
+    def studies(self) -> PaginationGenerator:
+        def pagination_method():
+            page_study = super(StudiesApi, self).list_studies(limit=20, offset=0)
+            assert isinstance(page_study, PageStudy)  # nosec
+            return page_study
+
+        return PaginationGenerator(
+            first_page_callback=pagination_method,
+            api_client=self.api_client,
+            base_url=self.api_client.configuration.host,
+            auth=self._auth,
+        )
+
+    def get_study_job_output_logfiles(self, study_id, job_id) -> Path:
+        return asyncio.run(
+            self.get_study_job_output_logfiles_async(study_id=study_id, job_id=job_id)
+        )
+
+    async def get_study_job_output_logfiles_async(self, study_id, job_id) -> Path:
+        """Download study logs. The log from each node will
+        appear as a file with the node's name in the directory"""
+        logs_map = super().get_study_job_output_logfile(study_id, job_id)
```

Comment:
Yes, but the automatically generated client uses threads. Here's the doc I find in the automatically generated client
```
        This method makes a synchronous HTTP request by default. To make an
        asynchronous HTTP request, please pass async_req=True
        >>> thread = api.get_study_job_output_logfile_with_http_info(study_id, job_id, async_req=True)
        >>> result = thread.get()
```
I don't think this is compatible with `asyncio` in the sense that I cannot `await` it.  Instead I could have wrapped it in a coroutine manually so I could await it. But I expect that the synchronous methods in the client are what will actually be called. I mainly did this async version to do the download of the files concurrently because there could be a lot.